java/src/client/turntracker/TurnTrackerController.java

```diff
@@ -29,7 +29,6 @@
 
     public TurnTrackerController(ITurnTrackerView view) {
         super(view);
-        initFromModel();
         GameManager.getGame().addObserver(this);
```

Comment:
Avoid the singleton `GameManager.getGame()`. It makes testing suck. Also, use `observeClientModel();` as the other controllers do for consistency, especially if we need to change how observing works.